packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/calendar.tests.js

```diff
@@ -4609,6 +4610,7 @@ QUnit.module('Navigation - click on other view cell', {
 
 QUnit.module('Aria accessibility', {
     beforeEach: function() {
+        this.DELAY = 5000;
```

Comment:
I'm pretty sure you can just create a constant at the top of the file right after the constant block with the css class names. There is no need to declare this variable here in before.

Reply:
[212f8b5](https://github.com/DevExpress/DevExtreme/pull/27898/commits/212f8b5f95a5f96df7644e0b287434c920ee75dc)